src/main/java/com/github/tomakehurst/wiremock/client/WireMock.java

```diff
@@ -561,6 +563,15 @@ public static MappingBuilder head(UrlPattern urlPattern) {
     return new BasicMappingBuilder(RequestMethod.HEAD, urlPattern);
   }
 
+  public static MappingBuilder getOrHead(String method, UrlPattern urlPattern) {
```

Comment:
I don't think this is the right approach - we should be building the stub mapping with the GET_OR_HEAD request method, then when matching it should match requests with either method and serve the body or not as appropriate.